### DISCUSSION TITLE
Documentation Improvement Ideas

### DISCUSSION THREAD

**Discussion by pkbullock** [pnp/pnpcore#158] [General] [answered]
I'm thinking about some content improvement ideas:

- Flatten documentation tab at top, to be the landing areas
- Consolidate contribution content into a section
- Add samples availability - using the "cards" that other projects have to showcase these

I've put this in a discussion, if anyone feels there is something lacking or missing that would really help them move forward, post here and we can look at getting these improved - of course, contributions also welcomed, and for those that want to help but don't know how, please contact us to help you get started.

**Your feedback is important to us - Sharing is Caring!**

**Comment by jansenbe** [comment, marked as answer]
Thanks @pkbullock !

It's not really clear to me what you mean with your first 2 points. I'm all in for improving the experience, not sure however to what extend this is possible with docfx as that's the tool that completely generates the documentation site on each code/doc change.

**Reply by pkbullock** [reply]
Sure, I am using the tool and I will work within its capabilities and source files. I have some ideas but need to test them e.g all samples have readme.md files, to refer to those in the generator. I'll do the improvements after issue #154
